lib/media/drm_engine.js

```diff
@@ -1907,13 +1915,22 @@ shaka.media.DrmEngine = class {
             },
           },
         };
-
+        // On some (Android) WebView environments, decodingInfo will
+        // not resolve or reject, at least if RESOURCE_PROTECTED_MEDIA_ID
+        // is not set.  This is a workaround for that issue.
+        const TIMEOUT_FOR_DECODING_INFO_IN_SECONDS = 1;
         const decodingInfo =
-            await navigator.mediaCapabilities.decodingInfo(decodingConfig);
+          await shaka.util.Functional.promiseWithTimeout(
+              TIMEOUT_FOR_DECODING_INFO_IN_SECONDS,
+              navigator.mediaCapabilities.decodingInfo(decodingConfig),
```

Comment:
We're doing this call primarily in `StreamUtils`, I guess you should add it there as well.